code/modules/mob/new_player/new_player_logout.dm

```diff
@@ -2,6 +2,7 @@
 	ready = FALSE
 	..()
 	if(!spawning)//Here so that if they are spawning and log out, the other procs can play out and they will have a mob to come back to.
+		mind.current = null // We best null their mind as well, otherwise /every/ single new player is going to explode the server a little more by logging out
```

Comment:
Just checking, is this suppose to be `mind`? The comment only says mind. I'm surprised new player mind's are having bodies attached to them.

Reply:
no mind current, but it's generally just refered to as a blanked mind